### PR TITLE
Update Reactor Netty version to 1.1.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by violetagg**
Reactor Netty QUIC version to 0.1.0-SNAPSHOT
Reactor BOM to 2022.0.0

### CODE REVIEW

**Review by simonbasle — APPROVED** [review]
LGTM, will need to be done after #2035 has been forward-merged into `main`

**Comment by violetagg** [comment]
@simonbasle @pderop Thanks for the review!

**Pull request merged**
